src/wr_logic_ai/nodes/state_machine/wait_for_user_input.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import rospy
+from wr_logic_ai.srv import EmptySrv, EmptySrvResponse
```

Comment:
Use standard empty service.

Reply:
Addressed in [277b00a](https://github.com/WisconsinRobotics/WRover_Software/pull/48/commits/277b00a47c8537aba6ba2aeeb6c09d9b5ea4d3eb).